exhale/graph.py

```diff
@@ -1639,7 +1639,7 @@ def fileRefDiscovery(self):
                         # see if this line represents the location tag
                         match = loc_regex.match(line)
                         if match is not None:
-                            f.location = match.groups()[0]
+                            f.location = os.path.normpath(match.groups()[0])
```

Comment:
Are you able to provide an example of why this is needed?

Reply:
Yes, ``location`` is tested against ``os.sep`` [here](https://github.com/svenevs/exhale/blob/master/exhale/graph.py#L1804) and then has ``os.path.dirname`` applied to it [here](https://github.com/svenevs/exhale/blob/master/exhale/graph.py#L1814. And then the file path is extracted from it [here](https://github.com/svenevs/exhale/blob/master/exhale/graph.py#L2020). Maybe it would actually be better to use ``os.path.normpath`` only there. Especially as this further investigation made me realize that my change may break things [here](https://github.com/svenevs/exhale/blob/master/exhale/graph.py#L1183).

Reply:
Hehehe.  And so it seems that you've discovered how ridiculous some parts of this framework are.  Don't stress, I actually installed Windows recently so I'll be able to test this weekend :)

I really need to refactor the graph module.  I've been putting it off because it's so damn daunting (and I wrote it all).